model/src/pyrenew/metaclass.py

```diff
@@ -412,3 +412,76 @@ def plot_posterior(
             draws_col=draws_col,
             obs_col=obs_col,
         )
+
+    def posterior_predictive(
+        self,
+        rng_key: ArrayLike | None = None,
+        *args,
```

Comment:
Slight preference for `*args` before any keyword args

https://docs.python.org/3/glossary.html#term-parameter
```suggestion
        *args,
        rng_key: ArrayLike | None = None,
```